src/index.js

```diff
@@ -188,6 +204,20 @@ module.exports = {
     return withRootUrl(rootUrl).servicesManifest();
   },
 
+  /**
+   * Method for generating links to task group or a task in Task Inspector
+   *
+   * @param rootUrl - string; rootUrl of the Taskcluster deployment,
+   *                  expected to be without a trailing slash
+   * @param taskGroupId - string
+   * @param taskId - string. Optional
+   *
+   * @returns string. URL to the task group (or the task if that parameter was also given) in Task Inspector
+   */
+  taskInspector(rootUrl, taskGroupId, taskId) {
+    return withRootUrl(rootUrl).taskInspector(taskGroupId, taskId);
```

Comment:
Here too.

Reply:
are you sure about this spot? This is how the rest of the methods work. I didn't intend this method to be different from the rest.

Reply:
Sorry, I got overly excited - indeed this is fine as is.